sphinx/source/docs/dev_guide/bokehjs.rst

```diff
@@ -130,23 +130,23 @@ The last to can be run with ``node make test:lib``. Unit and integration tests a
 run in a web browser (see requirements), which is started automatically with the
```

Comment:
I was not able to figure out what is meant by "The last to can be run" (line 129). Maybe "The last test can be run"? Or "The last one can be run"?

Reply:
It should be "The last two (commands) can be run (...)`.

Reply:
That makes a lot more sense, thank you!